### PR TITLE
Lighting

### PR DESCRIPTION
**Pull request by Pairjax**
- Lighting works as intended
- Need to account for out-of-date or wrong parts of the code
- Only need to keep changes in index, frag shader, and lighting.js

### CODE REVIEW

**Review by TheIceCreamBear — APPROVED** [review]

File: src/lighting.js [hunk 1]
```diff
@@ -3,55 +3,29 @@ import { Vector3 } from 'three';
...
+  getUniforms() {
+    return {
+      light: this.light,
+      ambient: this.ambient
+    };
+  }
 
-gl.uniformMatrix(programInfo.uniformLocations.normalMatrix, false, normalMatrix);
+  setDirection(x, y, z) {
+    this.light = (x, y, z);
```
Comment (ctcuff) [inline src/lighting.js:23]: Should be `new Vector3(x, y, z)` here

**Review by ctcuff — APPROVED** [review]
Small fix below but otherwise LGTM

File: src/lighting.js (hunk 1 above)
Comment (TheIceCreamBear) [reply src/lighting.js]: Resolved

**Review by Armintron — APPROVED** [review]
Very cool. Great work!

**Pull request merged**